### PR TITLE
Adding new tokens on Bifrost Polkadot

### PR DESCRIPTION
**Pull request by stepanLav**
GLMR:
Screenshot 2022-11-18 at 13 21 13
---
Screenshot 2022-11-18 at 14 05 22

DOT:
Screenshot 2022-11-18 at 14 01 32
---
Screenshot 2022-11-18 at 14 05 41

### CODE REVIEW

File: chains/v5/chains_dev.json [hunk 1]
```diff
@@ -4319,6 +4319,34 @@
                 "precision": 12,
                 "priceId": "bifrost-native-coin",
                 "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Bifrost.svg"
+            },
+            {
+                "assetId": 1,
+                "symbol": "GLMR",
+                "precision": 18,
```
Comment (leohar) [inline chains/v5/chains_dev.json:4326]: price id can be added

File: chains/v5/chains_dev.json [hunk 2]
```diff
@@ -4319,6 +4319,34 @@
...
+                    "currencyIdScale": "0x0801",
+                    "currencyIdType": "node_primitives.currency.CurrencyId",
+                    "existentialDeposit": "1000000000000",
+                    "transfersEnabled": true
+                }
+            },
+            {
+                "assetId": 2,
+                "symbol": "DOT",
+                "precision": 10,
```
Comment (leohar) [inline chains/v5/chains_dev.json:4340]: same here

File: chains/v5/chains_dev.json (hunk 1 above)
Comment (stepanLav) [reply chains/v5/chains_dev.json:4326]: I've added it bellow, but looks like it is better to have it after precision

Comment (leohar) [reply chains/v5/chains_dev.json:4326]: yeah, sorry didn't noticed

**Review by leohar — APPROVED** [review]

**Pull request merged**